apps/ui/lib/usecases/PhoneNumberValidation.mjs

```diff
@@ -0,0 +1,22 @@
+import UseCaseBase from './UseCaseBase.mjs';
+import { phone } from 'phone';
+import { COUNTRIES } from '../system/index.mjs';
+
+export class PhoneNumberValidation extends UseCaseBase {
+    static validationRules = {
+        phone: ['string'],
+    };
+
+    static isValid(phoneNumber, countries) {
```

Comment:
should not be as static in the use case. better place to the separate model, or it could be added custom rule for livr validator. Now I think this is a better approach.